### ISSUE TITLE
Checkbox focus state is offset

### ISSUE THREAD

**Issue by colbyfayock** [kognise/water.css#47]
Hey there, I noticed in your demo page the checkbox focus state is skewed. Just a heads up :)

I'm on Chrome Canary Version 75.0.3764.0 (Official Build) canary (64-bit)

![Screen Capture on 2019-04-14 at 10-01-22](https://user-images.githubusercontent.com/1045274/56093918-59710900-5e9c-11e9-9593-00cab683e997.gif)

**Comment by kylejrp** [comment]
@colbyfayock Are you on a Mac?

I'm getting a similarly weird offset on Chrome 73.0.3683.103 for Windows 10:
![image](https://user-images.githubusercontent.com/6742479/56096968-5a596900-5ea3-11e9-92ae-982533ff2ac3.png)

**Comment by colbyfayock** [comment]
yup! on a mac. yours looks more like a default browser focus outline 🤔where mine looks custom (it's a circle), though I'm unsure of the specifics beyond the following being computed for a custom focus state

![image](https://user-images.githubusercontent.com/1045274/56099305-2862f980-5ed9-11e9-9616-d17ea3bc009d.png)

**Comment by colbyfayock** [comment]
could strip the styling similar to the other inputs and do some kind of pseudo for the check 🤷‍♂️ 

![image](https://user-images.githubusercontent.com/1045274/56099404-76c4c800-5eda-11e9-907a-43b1b61e10f2.png)

**Comment by kognise** [comment]
Hmm - I was trying to avoid using custom checkboxes/radio buttons but if we're gonna do it we need to make it blend in as much as possible. Specifically, what should we use for the actual check while keeping download size? I feel like there are a billion different options.

On an unrelated note, in @colbyfayock's screenshot I noticed there's quite a lot less padding than I'm used to seeing with Water.css. Could this be a problem?

![](https://please.dont-hack.me/6fph0so4b2qgq.png)
![](https://please.dont-hack.me/asnnm8od0u174.png)

I'll try to write some code later, I'm unable to right now.

**Comment by colbyfayock** [comment]
that was just from me messing around in dev tools

you should be able to create a check, dot, or something using HTML characters (like the bullet I used in my screenshot) or CSS

**Comment by kognise** [comment]
I found [this](https://www.alt-codes.net/check-mark-symbols.php) with some check mark characters to try. This can't be too hard to implement, but should we implement custom radios as well?

Also, I **swear** at one point there were radios in the demo!

**Comment by colbyfayock** [comment]
yeah, I'd personally recommend the radio and checkbox have the same base UI with the exception that a radio is a circle and a checkbox is a square to show the intent

**Comment by kognise** [comment]
:+1:

**Comment by kognise** [comment]
Here's what I came up with. I need to add padding underneath the checkbox.

![image](https://user-images.githubusercontent.com/42556441/56379149-0a4d1000-61dd-11e9-9697-228e9c069bca.png)
